cmd/auto-updater/main.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/internal/sync2"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "auto-updater",
+		Short: "auto-updater for storage node",
+	}
+	runCmd = &cobra.Command{
+		Use:   "run",
+		Short: "Run the auto updater for storage node",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			err = cmdRun(cmd, args)
+			if err != nil {
+				fmt.Println(err)
+				os.Exit(1)
+			}
+			return nil
+		},
+	}
+
+	interval   string
+	versionURL string
+	binaryPath string
+)
+
+func init() {
+	rootCmd.AddCommand(runCmd)
+
+	runCmd.Flags().StringVar(&interval, "interval", "05s", "interval for checking the new version")
```

Comment:
Let's set the default to `6h` so it matches our current recommendation for watchtower.